smart-contracts/contracts/UnlockTestV2.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity 0.4.24;
+
+/**
+ * @title Example upgrade of the Unlock contract
+ * @author Julien Genestoux (unlock-protocol.com)
+ * @author HardlyDifficult
+ * 
+ * This inherits from the previous version, adds data and modifies logic.
+ */
+
+import "./Unlock.sol";
+
+
+contract UnlockTestV2 is Unlock {
```

Comment:
Ha! I think this is the best way to do it? Each new version `is` the previous version?

Reply:
That's a style question. The blog post you originally shared suggested this approach. It would make it easier to ensure the data is correct.

Reply:
Correct. I think we'll use that approach.